pyannote/audio/pipeline/speech_turn_segmentation.py

```diff
@@ -90,10 +90,14 @@ def __init__(self, sad_scores: Optional[Union[Path, str]] = None,
                 scores=self.sad_scores)
 
         self.scd_scores = scd_scores
-        self.speaker_change_detection = SpeakerChangeDetection(
-            scores=self.scd_scores)
-
-        self.non_speech = non_speech
+        if self.scd_scores == 'oracle':
+            self.non_speech = False
+            self.speaker_change_detection = OracleSpeechTurnSegmentation()
```

Comment:
`OracleSpeechTurnSegmentation` is **not** a speaker change detection replacement.
It is supposed to replace the combination of oracle VAD and oracle SCD.

Reply:
Oops, okay, I misunderstood your e-mail, sorry about that.
I fixed it.